merkle/objhasher/objhasher.go

```diff
@@ -16,42 +16,29 @@
 package objhasher
 
 import (
-	"crypto/sha256" // Use SHA256 to match ObjectHash.
+	"crypto"
 
 	"github.com/benlaurie/objecthash/go/objecthash"
+	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/merkle/maphasher"
```

Comment:
What do you advise? 
- `maphasher` currently contains a strict superset of `rfc6962`, so it can be used as both a log and a map. The only difference between the two is how `EmptyRoot` is computed.  However, the map code does not call this function, so I could set `EmptyRoot` to return the log's answer (`SHA256(nil)`)
- I could also create 
   - `objecthasher.NewLog()` and `objecthasher.NewMap()`
   - `logobjecthasher.New()` and `mapobjecthasher.New()`